xtask/src/release.rs

```diff
@@ -0,0 +1,162 @@
+use std::path::Path;
+
+use itertools::Itertools;
+use serde::Deserialize;
+use serde_json::json;
+use toml::from_str as from_toml_str;
+use xshell::{cmd, pushd, read_file};
+
+use crate::{config, flags, project_root, Result};
+
+const GITHUB_API_RELEASES: &str = "https://api.github.com/repos/zecakeh/ruma/releases";
```

Comment:
Looks like you accidentally committed what you wrote for testing.

Reply:
Sorry about that